plugins/inputs/snmp/snmp.go

```diff
@@ -10,6 +10,7 @@ import (
 	"math"
 	"net"
 	"os/exec"
+	"regexp"
 	"strconv"
 	"strings"
 	"sync"
```

Comment:
Is the readme file upto date now to reflect the new configuration options added?

Reply:
Sure, will update Readme.